libyang/schema.py

```diff
@@ -428,6 +436,14 @@ def module(self) -> Module:
             raise self.context.error("cannot get module")
         return Module(self.context, self.cdata_def.module)
 
+    def parent_node(self) -> Optional["SNode"]:
```

Comment:
Is this related to the extension plugin?

Reply:
Yes, but indirectly. Same as for parsed extension explained above, this one is for compiled extension